make/photon/jobservice/start.sh

```diff
@@ -1,7 +1,4 @@
 #!/bin/sh
-if [ -d /etc/jobservice/ ]; then
-    chown -R 10000:10000 /etc/jobservice/ 
-fi
 if [ -d /var/log/jobs ]; then
     chown -R 10000:10000 /var/log/jobs/
```

Comment:
shall we put this 'chown' into prepare? and set the dir mode for "/data/job_logs" to 700?
/data/job_logs:/var/log/jobs